repos/fdbt-site/tests/pages/api/returnService.test.ts

```diff
@@ -51,11 +73,47 @@ describe('returnService', () => {
             uuid: {},
             mockWriteHeadFn: writeHeadMock,
         });
+
         await returnService(req, res);
 
-        expect(writeHeadMock).toBeCalledWith(302, {
-            Location: '/error',
+        expect(redirectToErrorSpy).toBeCalledWith(
+            res,
+            'There was a problem selecting the additional return service:',
+            'api.returnService',
+            new Error('Could not find the ticket for which the service needs to be added.'),
+        );
+    });
+
+    it('should return 302 redirect to /error when the ticket being edited is a return ticket', async () => {
```

Comment:
should return 302 redirect to /error when the ticket being edited is not a return ticket

Reply:
fixed